lib/fortune.js

```diff
@@ -252,6 +253,14 @@ Fortune.prototype._exposeResourceDefinitions = function() {
   });
 };
 
+Fortune.prototype.customType = function(name, schema, options, schemaCallback) {
```

Comment:
Not sure if Mixed would really work here. If mongoose doesn't cast types for us (and it doesn't for Mixed type) we could get inconsistent data types.